bql/semantic/semantic.go

```diff
@@ -609,6 +610,11 @@ func (s *Statement) WorkingClause() *GraphClause {
 	return s.workingClause
 }
 
+// WorkingFilter returns the current working filter.
```

Comment:
In fact, at this moment it is not exactly being **applied** to the `Statement`.
`workingFilter` here is just like `workingClause` in the function above: it represents the current `FilterClause` struct that is being filled/constructed by the `WhereFilterClauseHook` before being added to the `filters` list of `Statement` by `AddWorkingFilterClause`.

The comment I wrote here is just like the comment for `workingClause` in the function above: "WorkingClause returns the current working graph clause", because they are both built following this same reasoning of hooks. I could add something like "... being constructed by hooks" to the comment, though maybe that would be a little out of scope for this `WorkingFilter` function. If you still prefer that I add something else to the comment, just lmk.